test/minitest/test_minitest_test.rb

```diff
@@ -146,6 +146,9 @@ def test_inherited_hook_plays_nice_with_others
 
 class TestMinitestRunner < MetaMetaMetaTestCase
   # do not parallelize this suite... it just can't handle it.
+  def self.test_order
+    :random
+  end
```

Comment:
I don't like these... Let me think about an alternative... but I'm thinking of something that runs `test_minitest_hell` entirely on its own.